examples/migrate_forseti/tutorial.md

```diff
@@ -160,7 +160,7 @@ to match the region where the CAI GCS bucket is deployed.
 Starting with Forseti Security 2.23, Terraform will manage your server
  configuration file for you.  Configuration options will now be input
  variables that are defined in the Terraform module. Available variables
- and their default values can be found [here](https://github.com/forseti-security/terraform-google-forseti/blob/module-release-5.0.0/variables.tf).
+ and their default values can be found [here](https://github.com/forseti-security/terraform-google-forseti/blob/module-release-5.2.0/variables.tf).
```

Comment:
Is this the right branch to refer to? I see there is also modulerelease520 that is referenced above.

Reply:
Good catch, thanks. Just committed a change. So many places where the release/branch need to be updated!